deployment/datastore/memory_datastore.go

```diff
@@ -100,18 +100,18 @@ func (s *MemoryDataStore[M]) Merge(other DataStore[M]) error {
 // It represents a sealed data store that cannot be modified further.
 var _ DataStore[DefaultMetadata] = &sealedMemoryDataStore[DefaultMetadata]{}
 
-type sealedMemoryDataStore[M Cloneable[M]] struct {
-	AddressRefStore *MemoryAddressRefStore          `json:"addressRefStore"`
-	MetadataStore   *MemoryContractMetadataStore[M] `json:"metadataStore"`
+type sealedMemoryDataStore[CM Cloneable[CM]] struct {
+	AddressRefStore       *MemoryAddressRefStore           `json:"addressRefStore"`
+	ContractMetadataStore *MemoryContractMetadataStore[CM] `json:"metadataStore"`
```

Comment:
rename the json tag?
```suggestion
	ContractMetadataStore *MemoryContractMetadataStore[CM] `json:"XXX"`
```

Reply:
done